src/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Game Leaderboard Projects</title>
+  </head>
+  <body class="container">
+    <header class="mt-5">
+      <h1 class="heading">Game Leaderboard</h1>
+    </header>
+    <main class="container d-flex flex-row justify-content-between mt-5">
+      <section class="col-6">
+        <div class="d-flex flex-row justify-content-between">
+          <h2>Recent Scores</h2>
+          <button class="btn btn-outline-secondary">Refresh</button>
+        </div>
+        <ul class="leaderBoard"></ul>
```

Comment:
- [ ] Consider adding some placeholder text for the scores so as to provide a visual for how the app will look  once it's functional

Your design
![image](https://github.com/Errytagedesign/gameLeaderboard/assets/81932478/fa90d747-9fe2-442b-90c0-47eab2aa4204)

Given wireframe
![image](https://github.com/Errytagedesign/gameLeaderboard/assets/81932478/58a955c7-296e-4cb5-b5c9-4fc54d307e23)